### PR TITLE
Add number_format function to all prices

### PR DESCRIPTION
**Pull request by ebisbe**
Now all prices shown by Cart and ItemCollection can be formated as desired. I've just wrapper for number_format() with a configuration file to configure it.

http://php.net/manual/en/function.number-format.php

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "require": {
         "php": ">=5.4.0",
         "illuminate/support": "5.0.*|5.1.*|5.2.*|5.3.*",
-        "illuminate/validation": "5.0.*|5.1.*|5.2.*|5.3.*"
+        "illuminate/validation": "5.0.*|5.1.*|5.2.*|5.3.4"
```
Comment (ebisbe) [inline composer.json]: There's a bug with versions > 5.3.4 https://github.com/laravel/framework/issues/16078

**Comment by ebisbe** [comment]
Hello @darryldecode,

Can you merge my PR??

**Comment by darryldecode** [comment]
👍

**Comment by it-can** [comment]
Can you please revert this or add a config option to disable this, my whole app is not working anymore...

**Comment by ebisbe** [comment]
You can define how many decimals you want to use. I have to write the documentation...

Add this to your .env file 
``SHOPPING_DECIMALS=2``
or publish config package parameters.

**Comment by it-can** [comment]
Let's continue this conversation in issue #74

**Pull request merged**